tests/test_cases/test_iteration_mixedlang/test_iteration.py

```diff
@@ -60,6 +60,22 @@ def test_drivers(dut):
         raise TestFailure("No drivers found for dut.i_verilog.uart1.uart_rx_1.rx_data")
 
 
+@cocotb.test(expect_fail=True)
+def test_loads(dut):
```

Comment:
```suggestion
async def test_loads(dut):
```

Reply:
This is an almost exact copy of the existing test for ``vpiDriver``, and this whole file is not yet converted to ``async`` syntax, so I'd do that in one go in another PR.

Reply:
I see the corresponding `test_drivers` test above it now that mostly identical. You could update that test as well in this PR with another commit? If not, I understand.